esx_service/utils/vmdk_utils.py

```diff
@@ -248,6 +248,15 @@ def get_vm_uuid_by_name(vm_name):
     except:
         return None
 
+def get_vm_name_by_uuid(vm_uuid):
+    """ Returns vm_name for given vm_uuid, or None """
+    si = vmdk_ops.get_si()
+    try:
+        vm = [d for d in si.content.rootFolder.childEntity[0].vmFolder.childEntity if d.config.uuid == vm_uuid]
+        return vm[0].config.name
```

Comment:
Made change to "create_vm" and "remove_vm" to avoid using functino "find_vm_by_name".